tests/cephadm/test_verify_upgrade_when_host_offline.py

```diff
@@ -0,0 +1,46 @@
+from ceph.ceph_admin.orch import Orch
```

Comment:
can we use - https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/orch/upgrade.py ?

Reply:
@vamahaja Yes, possible, but here it is taken from the existing upgrade test script that we are using to make sure that this issue is captured there or not. If required we can move it out to the new approach, but since its a negative scenario, do you suggest doing it?